docs/CLI.md

```diff
@@ -41,6 +41,13 @@ Tasks and variants can also be specified using the regex variants(-rv) and regex
 evergreen patch --regex_variants "enterprise.*" --rt "test-.*"
 ```
 
+Tasks and variants must _both_ be specified:
```

Comment:
Can we say something like "When specifying tasks and variants" or something like that at the beginning? Because it doesn't have to be specified, they can specify nothing and just do `evergreen patch -u`.